PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/ControlScript.rbxmx

```diff
@@ -320,11 +325,6 @@ LocalPlayer.CharacterAdded:connect(function(character)
 		ControlState:SwitchTo(lastControlState)
 	end
 end)
-
-LocalPlayer.CharacterRemoving:connect(function(character)
```

Comment:
if the character is removed, do we still show controls on the screen? It seems like we might want to keep this code as well as add it to characteradded